src/dialog/snippets/dialog.html

```diff
@@ -0,0 +1,37 @@
+<style>
+  .demo-dialog .mdl-dialog {
+    width: 1024px;
+  }
+</style>
+
+<div class="mdl-dialog mdl-transition-in" aria-label="Mango (Fruit)">
+  <div class="mdl-dialog-content" style="max-width:800px;max-height:810px; ">
```

Comment:
Why inline styles definition is necessary here? Could you move it either to example styles or to dialog.scss?